src/crunker.ts

```diff
@@ -330,6 +338,19 @@ export default class Crunker {
     return dataview;
   }
 
+  /**
+   * Copies data from buffer to DateView
+   *
+   * @internal
+   */
+  private _copyFloat32(dataview: DataView, buffer: Float32Array, offset: number): DataView {
```

Comment:
`nitpick`: I think it would be best to make sure this method and the `_floatTo16BitPCM` method and TSDoc are kept consistent since they are closely related in functionality.

Perhaps rename `_copyFloat32` or rename `_floatTo16BitPCM` so they convey the purpose a bit more. Will leave it with you what that name could be.